### DISCUSSION TITLE
Hey all

### DISCUSSION THREAD

**Discussion by ManishBisht777** [FrontendFreaks/Official-Website#95] [General]
Hey all how y'all doing and what your plans for hacktober fest?

did you plan to continue even after hacktober is finished?

**Comment by Vishal-raj-1** [comment]
Hey @ManishBisht777 I have lots of Idea that we can implement on this website.

**Reply by ManishBisht777** [reply]
Cool Lets start taking them one by one

**Reply by sridharrajaram** [reply]
@ManishBisht777 can you check the pr #220 and comment is it ok. Or shall I more semantics.

**Comment by utkarsh0903** [comment]
Beginning my open source contribution journey, starting with hacktober fest.

**Reply by ManishBisht777** [reply]
best of luck for that @utkarsh0903 :)
